src/Plugin.php

```diff
@@ -301,6 +302,12 @@ public static function wp_enqueue_scripts() {
     ]);
   }
 
+  public static function wp_enqueue_elementor_script() {
+    $git_version = static::getGitVersion();
+    $suffix = defined('SCRIPT_DEBUG') && SCRIPT_DEBUG ? '' : '.min';
+    wp_enqueue_script(Plugin::PREFIX, static::getBaseUrl() . '/dist/scripts/vendor/elementor' . $suffix . '.js', ['jquery'], $git_version, TRUE);
```

Comment:
The first parameter is the internal identifier of the script, which allows other scripts to use it as a dependency (in the same way you're specifying "jquery" here). The identifier must be unique for each script; otherwise WP assumes that the script has been loaded already.
```suggestion
    wp_enqueue_script(Plugin::PREFIX . '/elementor', static::getBaseUrl() . '/dist/scripts/vendor/elementor' . $suffix . '.js', ['jquery'], $git_version, TRUE);
```

Reply:
So I think something is wrong with the compiling still because this only works without the `$suffix`

Reply:
Okay, please remove `$suffix` for now.  The `.min.js` file is only compiled if `gulp` is invoked with the `--production` flag.  Most likely the `npm run build` command was not updated for this plugin yet.  Your branch only contains the unminified file, too.  This is very minor, the script very small.  We can improve the frontend performance later.

Reply:
Ok removed.